src/planscape/rscripts/forsys.R

```diff
@@ -174,12 +174,14 @@ get_stand_metrics <- function(
   query <- glue_sql(
     "SELECT
       stand_id,
-      {`metric_column`} as {`condition_name`}
+      CASE
+        WHEN {`metric_column`} IS NULL THEN 0
+        ELSE {`metric_column`}
+      END as {`condition_name`}
      FROM stands_standmetric
      WHERE
        condition_id = {condition_id} AND
-       stand_id IN ({stand_ids*}) AND
-       {`metric_column`} is NOT NULL",
+       stand_id IN ({stand_ids*})",
```

Comment:
nit: Have you considered using something like `COALESCE(metric_column, 0) AS condition_name`?
I think it's generally tidier to manage than CASE/WHEN and arguably faster if there's a ton of data.